test/tests/tests.vcxproj

```diff
@@ -77,4 +78,4 @@
     <ClCompile Include="TileElements.cpp" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Can you revert this new line change.

Reply:
I'm trying hard... I've tried visual studio and vim. Both insert this change automatically. :(
I'll keep trying.

Reply:
`:set nofixendofline` did the trick